requirements.txt

```diff
@@ -1,3 +1,3 @@
 psutil>=3.2.2
-signalfx>=1.0.1
+signalfx>=1.0.4
```

Comment:
@mdubbyap @ramjothikumar 
For the purposes of packaging and distribution of the plugin, leaving the requirement at ">=1.0.1" would not be a big deal.  At packaging time we would use pip and pull the latest version of the signalfx library and get v1.0.4.  

The only reason to increment the requirements.txt file is for people who get this directly from github.  If they have the signalfx python library installed already via pip at ... v 1.0.3, then when they install this requirements file the latest version of the library will not be pulled.  The requirement would technically be satisfied.

Because the changes to the library were a response to an issue observed in the plugin, I think it's a good idea to increment this requirement.